### PR TITLE
Update/web blue dot v4

### PR DESCRIPTION
**Pull request by JG1995**
blue dot updated for v4 of web

### CODE REVIEW

File: src/web/v4/guides/show-users-position.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+MapsIndoors supports both Google Maps and MapBox, and the methods for each vary slightly. Both still revolve around `PositionControl`.
+
+<mi-tabs>
+<mi-tab label="Google Maps" tab-for="google-maps"></mi-tab>
+<mi-tab label="Mapbox" tab-for="mapbox"></mi-tab>
+<mi-tab-panel id="google-maps">
+
+```js
+//GOOGLE MAP
```
Comment (eNepper) [inline src/web/v4/guides/show-users-position.md:54]:
```suggestion
```

File: src/web/v4/guides/show-users-position.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+});
+// Add the element now holding position control to your map
+googleMapsInstance.controls[google.maps.ControlPosition.RIGHT_BOTTOM].push(positionControlElement);
+```
+
+</mi-tab-panel>
+<mi-tab-panel id="mapbox">
+
+```js
+//MAPBOX
```
Comment (eNepper) [inline src/web/v4/guides/show-users-position.md:75]:
```suggestion
```

File: src/web/v4/guides/show-users-position.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+eleventyNavigation:
+  parent: web-v4-guides
+  key: Showing the user's location aka. Blue Dot
+---
+
+## Overview
+
+In this guide, you will learn how to show a dot on the map, representing the user's current location.
+
+The JSFiddle example below draws a MapsIndoors map, and adds a position control. Whenever position is received or updated, the map is always panned to that position.
```
Comment (eNepper) [inline src/web/v4/guides/show-users-position.md:12]: > Whenever position is received or updated, the map is always panned to that position.

It is somewhat only half the truth. If the user pans the map will say there and only center on the current position when the position control is clicked again.

Comment (eNepper) [inline src/web/v4/guides/show-users-position.md:12]:
```suggestion
The JSFiddle example below draws a MapsIndoors map, and adds a position control. Whenever a position is received or updated, the map is always panned to that position.
```

**Review by eNepper — COMMENTED** [review]
A couple of small things

File: src/web/v4/guides/show-users-position.md (hunk 3 above)
Comment (JG1995) [reply src/web/v4/guides/show-users-position.md]: comitted a fix to this

**Review by eNepper — APPROVED** [review]

**Pull request merged**